docs/rules/space-before-blocks.md

```diff
@@ -29,7 +29,7 @@ The default is `"always"`.
 Examples of **incorrect** code for this rule with the "always" option:
 
 ```js
-/*eslint space-before-blocks: "error"*/
+/*eslint space-before-blocks: "always"*/
```

Comment:
I think it's working as expected - the default is "always"

Reply:
the Example block is about 'always' but inside block 'error'. it's just wording correction

Reply:
the "error/warn/off" is a level, see the docs: https://eslint.org/docs/user-guide/configuring/rules#configuring-rules

Reply:
ah, my bad)